### ISSUE TITLE
Support for other cache backends like Memcached

### ISSUE THREAD

**Issue by asdrubalivan** [bolt/bolt#2860]
This is something I really would like to see in Bolt CMS (Maybe for the version 2.2). I've been using Yii for some years and I noticed they use a superclass for cache backends and then this class is inherited by other subclasses with each one representing a different backend. I consider this could be a good option, but since I don't have too much experience on Silex I think you guys will know better.

What are your thoughts?

Cheers, and thanks for creating this amazing CMS
Asdrúbal

**Comment by CarsonF** [comment]
Yeah I definitely want to do this. We've talked about it on IRC too. We just need to refactor our filesystem stuff out and pass a Cache interface to our Cache class. Shouldn't be too bad.

**Comment by SahAssar** [comment]
@CarsonF I'm really sorry to do this again (nagging on old issues), but IIRC you had an update on this?

**Comment by GawainLynch** [comment]
It's a time thing… I made a bunch of changes to the cache provider during the v3 extensions work to restore the proper interface… but given we're aiming to go beta on v3 soon, it's going to have to wait until a later 3.x to be properly fleshed out as a supported feature.

**Comment by GawainLynch** [comment]
@CarsonF and I have come to an agreement on approach here and we'll probably RFC & then PR for 3.4. The job isn't massive, but does require some more work in, and about to land in, 3.3 & master.

Bumping to 3.4.

**Comment by GawainLynch** [comment]
Bumping to 4.0

**Comment by SahAssar** [comment]
Still tagged for 4.0, so keeping open.

**Comment by GawainLynch** [comment]
Already implemented in the Flex banch
